src/auth/SchemeConfig.cc

```diff
@@ -131,6 +131,10 @@ Auth::SchemeConfig::parse(Auth::SchemeConfig * scheme, int, char *param_str)
         parse_onoff(&keep_alive);
     } else if (strcmp(param_str, "utf8") == 0) {
         parse_onoff(&utf8);
+    } else if (strcmp(param_str, "credentialsttl") == 0) {
+        time_t tmp = 0;
+        parse_time_t(&tmp);
+        credentialsTtl = Auth::Ttl(tmp);
```

Comment:
Here and elsewhere, please remove `Auth::` from the new code inside the Auth namespace.